src/session.ts

```diff
@@ -127,7 +127,8 @@ export class SessionManager implements Middleware {
             let powerShellExeDetails;
             if (this.sessionSettings.powerShellDefaultVersion) {
                 for (const details of this.powershellExeFinder.enumeratePowerShellInstallations()) {
-                    if (details.displayName === this.sessionSettings.powerShellDefaultVersion) {
+                    const wantedName = this.sessionSettings.powerShellDefaultVersion;
+                    if (wantedName.localeCompare(details.displayName, undefined, { sensitivity: "accent" }) === 0) {
```

Comment:
What's the undefined?

Reply:
[The `locales` parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare). Taken from [this SO answer](https://stackoverflow.com/a/2140723). There doesn't seem to be a more elegant way to do it unfortunately.

Reply:
Can you maybe throw that link to SO here in a comment?

Reply:
Ehhh or you don't have to. It's not that important

Reply:
Done!